src/db/awsES/locations-dao.js

```diff
@@ -0,0 +1,191 @@
+import _ from 'lodash';
+import aws from 'aws-sdk';
+import config from 'config';
+import connectionClass from 'http-aws-es';
+import elasticsearch from 'elasticsearch';
+import esb from 'elastic-builder';
+
+import { parseQuery } from 'utils/parse-query';
+
+const {
+  domain,
+  accessKeyId,
+  secretAccessKey,
+} = config.get('dataSources.awsES');
+
+/**
+ * Parses query parameters and generates an elasticsearch query body
+ *
+ * @param {object} queryParams Query paramaters from GET /locations request
+ * @returns {object} Elasticsearch query body
+ */
+const buildQueryBody = (queryParams) => {
+  const parsedParams = parseQuery(queryParams);
+  let q = esb.boolQuery();
+  if (parsedParams.search !== undefined) {
+    q = esb.multiMatchQuery([
+      'attributes.name',
+      'attributes.arcgisAbbreviation',
+      'attributes.bannerAbbreviation',
+    ], parsedParams.search);
+  }
+
+  if (parsedParams.name !== undefined) {
+    if (parsedParams.name.operator === 'fuzzy') {
+      q.must(esb.fuzzyQuery('attributes.name', parsedParams.name.value));
+    } else {
+      q.must(esb.termQuery('attributes.name.keyword', parsedParams.name));
+    }
+  }
+
+  if (parsedParams.type !== undefined) {
+    q.must(esb.matchQuery('attributes.type', parsedParams.type[0]));
+  }
+
+  if (parsedParams.hasGiRestroom !== undefined) {
+    if (parsedParams.hasGiRestroom) {
+      q.must(esb.rangeQuery('attributes.girCount').gt(0));
+    } else {
+      q.must(esb.matchQuery('attributes.girCount', 0));
+    }
+  }
+
+  if (parsedParams.adaParkingSpaceCount && parsedParams.adaParkingSpaceCount.operator === '>=') {
+    q.must(esb.rangeQuery('attributes.adaParkingSpaceCount')
+      .gte(parsedParams.adaParkingSpaceCount.value));
```

Comment:
Is it possible to align `.gte` with `.must`?

Reply:
Not without the linter whining: ` 53:1  error  Expected indentation of 6 spaces but found 5  indent`. I could get them to match up if `q` was a character longer

Reply:
Oh ok, no worries then.